docs/reference.cue

```diff
@@ -109,15 +109,15 @@ _values: {
 			}
 
 			if Kind != "source" {
-				input: #Event | [#Event, ...]
+				input: #Event | [...#Event]
 			}
 
 			if Kind == "sink" {
 				output: string
 			}
 
 			if Kind != "sink" {
-				output: #Event | [#Event, ...] | null
+				output: #Event | [...#Event] | null
```

Comment:
These should actually be `[#Event, ...#event]` instead. It'll require at least one item, which was the intent.